rust/src/importer.rs

```diff
@@ -56,17 +92,29 @@ impl RpmImporter {
         }
     }
 
-    /// Format tmpfiles.d lines for `/var/lib/<foo>` entries.
-    pub fn varlib_tmpfiles_symlinks(&self) -> Vec<String> {
-        let mut tmpfiles_lines = Vec::with_capacity(self.varlib_direntries.len());
+    /// Format tmpfiles.d lines for symlinked entries.
+    // NOTE(lucab): destinations (dirname) can't be quoted as systemd just
+    // parses the remainder of the line, and doesn't expand quotes.
+    pub fn tmpfiles_symlink_entries(&self) -> Vec<String> {
+        let mut tmpfiles_lines =
+            Vec::with_capacity(self.varlib_direntries.len() + self.opt_direntries.len());
+
+        // /opt/ symlinks
+        for dirname in &self.opt_direntries {
+            let linkpath = format!("/opt/{}", dirname);
+            let quoted = crate::maybe_shell_quote(&linkpath);
```

Comment:
Minor nit but why not fold this as: `let quoted = crate::maybe_shell_quote(&format!("/opt/{}", dirname));` ? I find it easier to read if there's only `dirname` and `quoted` in scope and I only need to think about those two.